core/frontend/src/components/autopilot/AutopilotManagerUpdater.ts

```diff
@@ -74,6 +75,8 @@ export async function fetchFirmwareInfo(): Promise<void> {
       url: `${autopilot.API_URL}/firmware_info`,
       timeout: 10000,
     })
+    // Version comes out as a string, let's turn it into a SemVer object
+    response.data.version = new SemVer(response.data.version)
```

Comment:
Is this the only place we need to make this conversion?

Reply:
it is the only one I found so far. I'm too deep in the params thing to look for others...

Reply:
I checked, and it is.